src/components/layout/MunicipalityLayout.tsx

```diff
@@ -76,8 +75,6 @@ function MunicipalityLayout(props: WithChildren) {
   }
 
   function handleMunicipalitySelect(region: TMunicipality[number]) {
```

Comment:
If the aforementioned change is made, then this can simply become:
```suggestion
  function handleMunicipalitySelect(region: TMunicipality) {
```